superdesk/notification.py

```diff
@@ -15,9 +15,9 @@
 import asyncio
 import websockets
 
-from flask import json
 from flask import current_app as app
 from datetime import datetime
+from bson.json_util import dumps
```

Comment:
You can use this `flask.json` with `json_serialize_datetime_objectId` 
see :
https://github.com/superdesk/superdesk-core/blob/master/superdesk/utils.py#L142
https://github.com/superdesk/superdesk/blob/master/server/apps/publish/formatters/aap_bulletinbuilder_formatter.py#L51